<%
cn.close();
}
catch(Exception ex)
{
	%>
	<%="Error:"+ex%>
	<%
}
%>
